### PR TITLE
chore: add security document

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

### Related changes

* PR to update `chaos-mesh/website`/`chaos-mesh/website-zh`:
* Need to update Chaos Dashboard component, related issue:
* Need to cheery-pick to the release branch

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

### Release note 

```release-note
Please add a release note.
If you don't think this PR needs a release note then fill it with None.
```

### CODE REVIEW

**Comment by milasuperstar** [comment]
LGTM

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Disclosure policy
+
+For known public security vulnerabilities, we will disclose the disclosure as soon as possible after receiving the report. Vulnerabilities discovered for the first time will be disclosed in accordance with the following process:
+
+1. The received security vulnerability report shall be handed over to the security team for follow-up coordination and repair work.
+2. After the vulnerability is confirmed, we will create a draft Security Advisory on Github that lists the details of the vulnerability.
+3. Invite related personnel to discuss the fix.
+4. Fork the temporary private repository on Github, and collaborate to fix the vulnerability.
+5. After the fix code is merged into all supported versions, the vulnerability will be publicly posted in the GitHub Advisory Database.
```
Comment (fewdan) [inline SECURITY.md:24]: change to "After the fixed code ....."  ?

**Review by dcalvin — APPROVED** [review]
LGTM

File: SECURITY.md (hunk 1 above)
Comment (dcalvin) [reply SECURITY.md]: Please fix this @cwen0

**Review by g1eny0ung — APPROVED** [review]
LGTM

**Pull request merged**